### PR TITLE
add mix-blend-mode=difference to high-contrast images in KeyboardHelp

### PR DESCRIPTION
**Pull request by beyackle2**
The mix-blend-mode CSS property is useful for enforcing that something show up black on a white background or white on a black background without having to know beforehand what color the background is. This change uses that property to make the Keyboard Help screen legible even in high-contrast themes that enforce a light background.

> Fixes [issue linked on ADO](https://msazure.visualstudio.com/CCI/_workitems/edit/15627465)

## Specific Changes

This adds `mix-blend-mode: difference` to the places in KeyboardHelp.tsx where we need an image to contrast against a possibly-unknown background color.

## -

-

-  [x] I have updated `CHANGELOG.md`
-  [ ] I have updated documentation

## Review Checklist

> This section is for contributors to review your work.

-  [ ] Accessibility reviewed (tab order, content readability, alt text, color contrast)
-  [ ] CSS styles reviewed (minimal rules, no `z-index`)

Dark background:
image
Light background:
image

### CODE REVIEW

File: packages/component/src/Transcript/KeyboardHelp.tsx [hunk 1]
```diff
@@ -111,6 +111,9 @@ const KeyboardHelp: FC<{}> = () => {
             // "focusable" attribute is only available in IE11 and "tabIndex={-1}" does not work.
             focusable={false}
             role="presentation"
+            style={{
```
Comment (compulim) [inline packages/component/src/Transcript/KeyboardHelp.tsx:114]: Could you cache the `style` props into a constant to reduce wasted render?

File: CHANGELOG.md [hunk 2]
```diff
@@ -22,6 +22,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+-  Keyboard help now shows images correctly in Windows 11's "desert" high-contrast mode, by [@beyackle2](https://github.com/beyackle2) in PR [#4531](https://github.com/microsoft/BotFramework-WebChat/pull/4531)
```
Comment (compulim) [inline CHANGELOG.md:25]:
```suggestion
### Fixed

-  Keyboard help now shows images correctly in Windows 11's "desert" high-contrast mode, by [@beyackle2](https://github.com/beyackle2) in PR [#4531](https://github.com/microsoft/BotFramework-WebChat/pull/4531)
```

**Review by compulim — CHANGES_REQUESTED** [review]
I think `.husky/pre-commit` is Git-ignored.

Also need tests. We have screenshot tests, but I am unsure how we could force high contrast mode in Linux Chrome. Maybe need to look at some Chrome options.

File: packages/component/src/Transcript/KeyboardHelp.tsx (hunk 1 above)
Comment (beyackle2) [reply packages/component/src/Transcript/KeyboardHelp.tsx:114]: Good idea.

Comment (beyackle2) [reply packages/component/src/Transcript/KeyboardHelp.tsx]: Fixed this. Still trying to figure out how to write a test for high-contrast mode.

**Pull request closed without merging**